modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/utils/api.ts

```diff
@@ -29,3 +29,11 @@ export function deleteConnection() {
 		method: 'DELETE',
 	});
 }
+
+export function fetchConnectionModal(propertyName: string) {
+	return new Promise((resolve) => {
+		setTimeout(() => {
+			resolve({connected: true, propertyName});
+		}, 2000);
+	});
+}
```

Comment:
Could you rebase your PR with portal master once this function doesn't exists anymore?

Reply:
@JLucasM could you please do that? Also, fix the conflicts please